modules/ebi-metagenomics/pimento/generatebcv/main.nf

```diff
@@ -38,44 +35,24 @@ process ASSESSMCPPROPORTIONS {
 
     if (strands == "") {
         """
-        touch ${prefix}_${var_region}_mcp_cons.tsv
+        touch ${assess_mcp_prop_prefix}_mcp_cons.tsv
 
         cat <<-END_VERSIONS > versions.yml
         "${task.process}":
-            mgnify-pipelines-toolkit: \$(get_mpt_version)
+            mi-pimento: \$( pimento --version | cut -d" " -f3 )
```

Comment:
Seqera AI is very against cut :D
`Using cut instead of sed: While cut can work, nf-core guidelines specifically recommend using sed for version extraction as it's more flexible and powerful.`
```
cat <<-END_VERSIONS > versions.yml
"${task.process}":
    pimento: \$( pimento --version | sed '1!d; s/.*version //; s/ .*//' )
END_VERSIONS
```
or
```
cat <<-END_VERSIONS > versions.yml
"${task.process}":
    pimento: \$( pimento --version | sed '1!d; s/.*version //; s/ .*//' || true )
END_VERSIONS
```
and maybe we should use `pimento` instead of `mi-pimento`

Reply:
the package is actually called mi-pimento on bioconda (pimento was already taken 😢 )... does that matter to us?

Reply:
ah, ok I see